packages/cli/src/oclif/commands/migrate.js

```diff
@@ -58,8 +62,12 @@ class MigrateCommand extends BaseCommand {
         `Starting migration from ${fromVersion} to ${toVersion} for ${percent}%`
       );
     }
+    if (percent) {
+      body.job.percent_human = percent;
```

Comment:
I believe this can also be included rigtht in the `job` above, since it's always defined (either from the default or from the user passing it explicitly)